pkg/cli/cli.go

```diff
@@ -55,6 +57,7 @@ var templates = map[string]templateInfo{
 
 const defaultOutputType = "changelog"
 const defaultRepositoryFilename = "suite.yml"
+const defaultReleasesDir = "releases"
```

Comment:
Should we update the readme with this as well? This means a user _must_ have a releases directory, however I think it might be better for that to be optional.

Reply:
I'll update the readme. I did think about your point raised here earlier but our progress forward is leveraged on using diffs between last releases and current suite version so the default of a "single suite.yml" file is not the expected workflow except when checking for unreleased changes (which we handle fine since it's in an if/else).

Reply:
What about the use case where it hasn't been ran before, and a previous version doesn't exist?

Reply:
Hmm.. maybe I can add one more switch for that.

Reply:
I decided to just better document how to do it in both readme and the help output (you set the releases dir to empty string)